.github/workflows/build.yml

```diff
@@ -1,12 +1,16 @@
+---
 name: build
 
 on:
+  workflow_dispatch:
+  pull_request:
   push:
     # Sequence of patterns matched against refs/heads
     branches:
       # Push events on the main branch
       - main
       - release/**
+      - releng-329
```

Comment:
Does L13 need to be removed before merge? I'm guessing this was just used for testing?

Reply:
Great catch - we can absolutely remove that 👍🏻